addons/l10n_uy/demo/demo_company.xml

```diff
@@ -3,11 +3,10 @@
     <record id="partner_demo_company_uy" model="res.partner">
         <field name="name">UY Company</field>
         <field name='l10n_latam_identification_type_id' ref='it_rut'/>
-        <field name="vat">UY130551600010</field>
+        <field name="vat">218296790015</field>
         <field name="street"/>
         <field name="city">Montevideo</field>
         <field name="country_id" ref="base.uy"/>
-        
```

Comment:
Please add here         `<field name='state_id' ref="base.state_uy_10"/>`
we will needed for EDI test